shell/client/admin/user-details.html

```diff
@@ -327,11 +327,16 @@
           <strong>Account deletion is permanent and irreversible.</strong>  Please be certain.
           The account will first be suspended for 7 days so that you have a chance to undo the
           deletion. At the end of the 7 days it will be deleted.
+
+        </p>
+        <p>
+          Please type "Yes, delete this account" below to confirm.
         </p>
+        <input class="confirm" type="text">
```

Comment:
Talked in person with @zarvox, and decided it wasn't worth it to wrap it in a form.